pandas/tests/io/test_parquet.py

```diff
@@ -437,6 +437,27 @@ def test_s3_roundtrip(self, df_compat, s3_resource, pa):
         check_round_trip(df_compat, pa,
                          path='s3://pandas-test/pyarrow.parquet')
 
+    @pytest.mark.xfail(
+        is_platform_windows() or is_platform_mac(),
```

Comment:
Not really. This feature is useful mostly when you have hundreds of large files.
Such deployments are almost 100% Linux. I've never seen such deployment on Windows or Mac.

I put it because couple lines above you have
```
@pytest.mark.xfail(is_platform_windows() or is_platform_mac(),
                   reason="reading pa metadata failing on Windows/mac")
def test_cross_engine_pa_fp(df_cross_compat, pa, fp):
```
The above comment suggests the features are less mature on Windows/Mac.
I do not have Windows or Mac so I can not test it/check it.

Anyone who needs it on Windows/Mac could validate it and correct.

Reply:
Our CI runs on Mac, Linux, and Windows.

Reply:
@JacekPliszka we have a test of cross compatibility of fastparquet and pyarrow. this is a very specific instance. this is not applicable to this PR